crates/nu-cli/src/format/table.rs

```diff
@@ -9,7 +9,7 @@ use textwrap::fill;
 use prettytable::format::{Alignment, FormatBuilder, LinePosition, LineSeparator};
 use prettytable::{color, Attr, Cell, Row, Table};
 
-type Entries = Vec<Vec<(String, &'static str)>>;
+type Entries = Vec<Vec<(String, String)>>;
```

Comment:
There are quite a few changes from `&'static str` to `String` that were done to keep Rustc happy, but I'm not convinced that all of them have to be Strings, some could probably be `&str` or maybe even kept at `&'static str` possibly (I'm not sure).